README.md

```diff
@@ -1,4 +1,4 @@
-ACL (Access Control List)
+acl
```

Comment:
For those of us who know what an ACL is, it's not a worry. For those that don't, the first question is what is it?